html/admin/js/connectbox-ui.js

```diff
@@ -208,7 +208,9 @@ var ConnectBoxApp = (function (ConnectBoxApp, $) {
 
     ConnectBoxApp.api.getProperty('channel', function (result, code, message) {
       if (result) {
-        $('#input_channel').val(result[0])
+        // the channel number is 1-based int, but the selectedIndex of the
+        //  HTMLSelectedElement is 0-based, so we subtract 1
+        $('#input_channel')[0].selectedIndex = result[0] - 1
```

Comment:
Here I believe you can actually just revert to the original code to set the value. See this jsfiddle: https://jsfiddle.net/7vc4cxqb/